scripts/installation/compile_MET_all.sh

```diff
@@ -128,18 +128,18 @@ COMPILER_VERSION=`echo $COMPILER | cut -d'_' -f2`
 
 echo "USE_MODULES = ${USE_MODULES}"
 
-if [ ${USE_MODULES} == "TRUE" ]; then 
+if [ ${USE_MODULES} = "TRUE" ]; then 
```

Comment:
Thank you for looking at this.  I thought the same thing too, but received a "[: ==: unary operator expected" message when running this code, which led me to this https://stackoverflow.com/questions/13617843/unary-operator-expected-error-in-bash-if-condition, where I noticed that only one "=" was being used for comparison, which led me to this
https://linuxize.com/post/how-to-compare-strings-in-bash/ which says:
```
Comparison operators are operators that compare values and return true or false. When comparing strings in Bash you can use the following operators:
string1 = string2 and string1 == string2 - The equality operator returns true if the operands are equal.
Use the = operator with the test [ command.
Use the == operator with the [[ command for pattern matching.
```
I am no bash expert, so I don't really know.  Just following the guidance on these webpages and wanting to do it correctly.  Thoughts and/or suggestions?

Reply:
Julie, thanks for clarifying. I see other instances where we use [[ "a" == "b" ]] syntax. To avoid confusion do you want to switch those comparisons to [ "a" = "b" ] as well? Or is that inviting problems?

Reply:
In the examples from that webpage it says to:
```
Use the = operator with the test [ command.
Use the == operator with the [[ command for pattern matching.
```
So it looks like I should use one "=" for comparisons, when we use one bracket and two "==" for comparson when we use two brackets.  I'm not sure when to use one vs. two brackets either.  It looks like we should use one for a "test" and two for "pattern matching".  Maybe I should go back and check all for tests vs. pattern matching?  Thoughts?

Reply:
I just approved the changes, as long as they actually work ;) Don't want to send us down an unnecessary rabbit hole hole here.

Reply:
Sounds good, John.  Thanks.  I'll be testing this when I start working on having MET compile with PGI v9 and for beta5, and can make additional updates as necessary in a future pull request.  I won't put this up on the website just yet and will likely wait until the official release.